packages/m2m-gateway/src/services/tenantService.ts

```diff
@@ -38,5 +40,57 @@ export function tenantServiceBuilder(clients: PagoPAInteropBeClients) {
         },
       };
     },
+    getCertifiedAttributes: async (
+      tenantId: TenantId,
+      { limit, offset }: m2mGatewayApi.GetCertifiedAttributesQueryParams,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.TenantCertifiedAttributes> => {
+      logger.info(`Retrieving tenant ${tenantId} certified attributes`);
+
+      const { data: tenant } =
+        await clients.tenantProcessClient.tenant.getTenant({
+          params: { id: tenantId },
+          headers,
+        });
+
+      const tenantCertifiedAttributes = tenant.attributes
+        .map((v) => v.certified)
+        .filter(isDefined);
+
+      const tenantCertifiedAttributeIds = tenantCertifiedAttributes.map(
+        (att) => att.id
+      );
+
+      const {
+        data: { results: certifiedAttributes, totalCount },
+      } = await clients.attributeProcessClient.getBulkedAttributes(
+        tenantCertifiedAttributeIds,
+        {
+          headers,
+          queries: {
+            offset,
+            limit,
+          },
+        }
+      );
+
+      const combinedAttributes = zipBy(
+        tenantCertifiedAttributes,
+        certifiedAttributes,
+        ({ id }) => id,
+        ({ id }) => id
+      );
+
+      return {
+        results: combinedAttributes.map((args) =>
+          toM2MGatewayApiTenantCertifiedAttribute(...args)
+        ),
```

Comment:
Minor: expliciting the params makes this more readable IMO 👇 

```suggestion
        results: combinedAttributes.map(([tenantCertifiedAttribute,  certifiedAttribute]) =>
          toM2MGatewayApiTenantCertifiedAttribute(tenantCertifiedAttribute,  certifiedAttribute)
        ),
```